main.qml

```diff
@@ -315,16 +315,37 @@ ApplicationWindow {
             webView.url = webView.mainUrl;
         }
     }
+    function injectJS() {
+        webView.webChannel.registerObject( 'transport', transport );
+        // Try-catch to be able to return the error as result, but still throw it in the client context
+        // so it can be caught and reported
+        var injectedJS = "try { initShellComm() } " +
+                "catch(e) { setTimeout(function() { throw e }); e.message || JSON.stringify(e) }";
+        webView.runJavaScript(injectedJS, function(err) {
+            if (!err) {
+                webView.tries = 0
+            } else {
+                errorDialog.text = "Error while applying shell JS." +
+                        " Please consider re-installing Stremio from https://www.stremio.com"
+                errorDialog.detailedText = err
+                errorDialog.visible = true
+
+                console.error(err)
+            }
+        });
+    }
 
     // We want to remove the splash after a minute
     Timer {
         id: removeSplashTimer
-        interval: 60000
+        interval: 90000
         running: true
         repeat: false
         onTriggered: function () {
+            webView.backgroundColor = "transparent"
             splashScreen.visible = false
             pulseOpacity.running = false
+            injectJS();
```

Comment:
use consistent code style.. as far as i can see  most of the code does not use semicolons

Reply:
Done